learning-pathways/galaxy_academy_2024.md

```diff
@@ -0,0 +1,55 @@
+---
+layout: learning-pathway
+
+title: Galaxy Academy 2024
+description: |
+  The Galaxy Academy is a online training event for Beginners as well as learners who would like to improve there Galaxy data analysis skills. Over the course of one week we will have a different topic and focus every day. 
+tags: [beginner]
+
+cover-image: path/to/image.png # optional cover image, defaults to GTN logo
+cover-image-alt: alt text for this image
+
+pathway:
+  - section: "Monday: Setting up Galaxy"
+    description: |
+      Today will be all about getting to know Galaxy and explore its potentials. Additionally you can learn about the foundation of data analysis. 
+    tutorials:
+      - name: galaxy-intro-short
+        topic: introduction
+      - name: galaxy-intro-101
+        topic: introduction
+      - name: quality-control
+        topic: sequence-analysis
+      - name: mapping
+        topic: sequence-analysis
+ 
+  - section: "Tuesday: Transcriptomics"
+    description: Today you will learn all about transcriptomics
+    tutorials:
+      - name: ref-based
+        topic: transcriptomics
+
+  - section: "Wednesday: Proteomics"
+    description: 
+    tutorials:
+      - name: introduction
+        topic: proteomics
+
+  - section: "Thursday: Microbiome"
+    description: 
+    tutorials:
+      - name: introduction
+        topic: microbiome
+
+  - section: "Friday: Grab bag"
+    description: | 
+      Here we have some additional topics. Please feel free to pick and choose the tutorials that are interesting for you.
+    tutorials:
+      - name: troubleshooting
+        topic: admin
```

Comment:
I guess this is just an example? Maybe a non-admin example could be better here :)